lib/colorls/core.rb

```diff
@@ -17,11 +17,15 @@ def initialize(input=nil, all: false, report: false, sort: false, show: false,
     end
 
     def ls
-      @contents = chunkify
-      @max_widths = @contents.transpose.map { |c| c.map(&:length).max }
-      @contents.each { |chunk| ls_line(chunk) }
-      print "\n"
-      display_report if @report
+      if @contents.empty?
+        print "it is empty baby \uf119\n".colorize(:blue)
+      else
```

Comment:
Oww I did not see it that way, It should be fixed now. My bad.

Alright It sould be done !